core/alarms/alarms.ts

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+import { cascade, type AlarmsCascade } from '../inputs/cascading-config'
+import { applyAlarmConfig } from '../inputs/function-config'
+import { type FunctionAlarmPropertiess, type Context } from './default-config-alarms'
+import type Template from 'cloudform-types/types/template'
+
+import createLambdaAlarms from './lambda'
+import createApiGatewayAlarms from './api-gateway'
+import createStatesAlarms from './step-functions'
+import createDynamoDbAlarms from './dynamodb'
+import createKinesisAlarms from './kinesis'
+import createSQSAlarms from './sqs'
+import createECSAlarms from './ecs'
+import createSNSAlarms from './sns'
+import createRuleAlarms from './eventbridge'
+import createALBAlarms from './alb'
+import createALBTargetAlarms from './alb-target-group'
+import createAppSyncAlarms from './appsync'
+import { type ResourceType } from './../cf-template'
+
+export default function addAlarms (AlarmProperties: AlarmsCascade, functionAlarmProperties: FunctionAlarmPropertiess, context: Context, compiledTemplate: Template, additionalResources: ResourceType = {}) {
```

Comment:
`AlarmProperties` can be camelCase (`alarmProperties`) and `AlarmCascade` can be named `AlarmProperties` or `AlarmConfig`. The properties can be cascaded but that is not something that needs to be reflected in the type name.